[2021-05-27 18:13:56,022] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: dag_transfers_scraping.load_transfers_CSV_to_S3 2021-05-26T02:00:00+00:00 [queued]>
[2021-05-27 18:13:56,095] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: dag_transfers_scraping.load_transfers_CSV_to_S3 2021-05-26T02:00:00+00:00 [queued]>
[2021-05-27 18:13:56,095] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-05-27 18:13:56,095] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-05-27 18:13:56,095] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-05-27 18:13:56,110] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): load_transfers_CSV_to_S3> on 2021-05-26T02:00:00+00:00
[2021-05-27 18:13:56,117] {standard_task_runner.py:52} INFO - Started process 97304 to run task
[2021-05-27 18:13:56,125] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag_transfers_scraping', 'load_transfers_CSV_to_S3', '2021-05-26T02:00:00+00:00', '--job-id', '102', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_transfers_scraping.py', '--cfg-path', '/tmp/tmpwjjzofem', '--error-file', '/tmp/tmpe41ewuuk']
[2021-05-27 18:13:56,127] {standard_task_runner.py:77} INFO - Job 102: Subtask load_transfers_CSV_to_S3
[2021-05-27 18:13:56,274] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_transfers_scraping.load_transfers_CSV_to_S3 2021-05-26T02:00:00+00:00 [running]> on host f7393dd572fa
[2021-05-27 18:13:56,382] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_transfers_scraping
AIRFLOW_CTX_TASK_ID=load_transfers_CSV_to_S3
AIRFLOW_CTX_EXECUTION_DATE=2021-05-26T02:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-26T02:00:00+00:00
[2021-05-27 18:13:56,388] {base_aws.py:368} INFO - Airflow Connection: aws_conn_id=aws_s3_***_user
[2021-05-27 18:13:56,437] {base_aws.py:170} INFO - Credentials retrieved from extra_config
[2021-05-27 18:13:56,438] {base_aws.py:87} INFO - Creating session with aws_access_key_id=AKIAZP4IG4HNCZJ3LJDL region_name=None
[2021-05-27 18:13:56,458] {base_aws.py:157} INFO - role_arn is None
[2021-05-27 18:13:57,830] {python.py:151} INFO - Done. Returned value was: None
[2021-05-27 18:13:57,859] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=dag_transfers_scraping, task_id=load_transfers_CSV_to_S3, execution_date=20210526T020000, start_date=20210527T181356, end_date=20210527T181357
[2021-05-27 18:13:57,910] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-27 18:13:57,934] {local_task_job.py:151} INFO - Task exited with return code 0
